webservices/args.py

```diff
@@ -223,6 +223,8 @@ def make_seek_args(field=fields.Int, description=None):
 efilings = {
     'file_number': fields.List(fields.Int, description=docs.FILE_NUMBER),
     'committee_id': fields.List(IStr, description=docs.COMMITTEE_ID),
+    'min_receipt_date': fields.DateTime(description='Selects all items received by FEC after this date and time'),
```

Comment:
For people who might read these descriptions would it be better to say 'date and/or time', or somehow convey that it doesn't have to be a time?